packages/browser/src/Widget/Server/SPARQL/View/SparqlQueryWindow.js

```diff
@@ -304,6 +265,50 @@ export class SparqlQueryWindow extends EventSender {
     });
   }
 
+  /**
+   *
+   * @param {string} id an identifier
+   * @returns {Array<Array<string>>}
+   */
+  getTransactionChain(id) {
+    const result = this.sparqlProvider.querySparqlEndpointService(
+      this.transactionChainQuery(id)
+    );
+    return result;
+  }
+
+  /**
+   * Get the CityGML 3.0 transaction chain query for as a string
```

Comment:
this class is about sparql is a citygml behavior is apropriate here ?

Reply:
@CorentinGaut this function is being moved to the `packages/browser/src/Widget/Server/SPARQL/View/SparqlQueryWindow.js` class see [here](https://github.com/VCityTeam/UD-Viz/pull/723/commits/3cadfd6b2479175714b33fc78f6e2ab91ec0b528)